pymc3/distributions/continuous.py

```diff
@@ -2844,7 +2832,6 @@ def logp(self, value):
             - 0.5 * at.log(nu * np.pi * sigma ** 2)
             - (nu + 1.0) / 2.0 * at.log1p(value ** 2 / (nu * sigma ** 2)),
             sigma > 0,
-            lam > 0,
```

Comment:
I'm a bit skeptical about this particular parametrization, since that required removing this condition over here.(I don't know if that particular condition is important or not, though)

Reply:
The lam sigma is a bit weird in that one can be converted from the other, even if the original is negative. You can obtain the lam again from sigma by  using the `get_tau_sigma` inside the `logp`. This way you can keep the lam check in the `bound`, but probably we should just open a separate issue to fix `get_tau_sigma`

Reply:
Ah, okay got it.